data_collection/gazette/spiders/ba/ba_antonio_cardoso.py

```diff
@@ -5,3 +5,4 @@ class BaAntonioCardosoSpider(DoemGazetteSpider):
     TERRITORY_ID = "2901700"
     name = "ba_antonio_cardoso"
     state_city_url_part = "ba/antoniocardoso"
+    start_date = date(2018, 1, 3)
```

Comment:
```suggestion
    start_date = date(2017, 1, 2)
```
Sei que no botão do site aparece só até 2018, porém se vc explorar a URL (https://doem.org.br/ba/antoniocardoso/diarios/{ANO}/) vai ver que abre pra mais diários! ;) 

https://doem.org.br/ba/antoniocardoso/diarios/2017/1